td/tracking/templates/tracking/event_form.html

```diff
@@ -209,3 +229,78 @@ <h2 class="panel-title">New Translation Event/Workshop</h2>
 	</div>
 
 {% endblock body%}
+
+{% block custom_bottom_script %}
+    <script src="http://malsup.github.com/jquery.form.js"></script>
+    <script>
+    	['translator', 'facilitator', 'material'].forEach(function(field) {
+	    	$('form').on('paste', 'input[name*="' + field + '"]', function(e) {
+	    		e.preventDefault();
+	    		var text = e.originalEvent.clipboardData.getData('Text').split(/[,\t\n]/);
+	    		var fields = $('input[type="text"][name*="' + field + '"]');
+	    		for (var i = 0; i < text.length - fields.length; i++) {
+	    			addInline(field);
+	    		}
+	    		$('input[type="text"][name*="' + field + '"]').each(function(index) {
+	    			this.value = text[index] || "";
+	    		});
+	    	});
+    	});
+
+
+		// ================
```

Comment:
The level of indentation looks off here - mixing of tabs/spaces or is it correct?